apps/api/src/app/shared/services/cache/cache.service.ts

```diff
@@ -0,0 +1,72 @@
+import Redis from 'ioredis';
+import { getRedisPrefix } from '@novu/shared';
+
+interface ICacheService {
+  set(key: string, value: string, options?: CachingConfig);
+  get(key: string);
+  del(key: string);
+  reset();
+}
+
+export type CachingConfig = {
+  ttl?: number;
+};
+
+export class CacheService {
+  private readonly client: Redis;
+  private readonly DEFAULT_TTL = 60;
```

Comment:
Is this seconds or milliseconds?

Reply:
Seconds, and it is more explained on the updateTtl function do you think it adding here seconds suffix will be better?

Reply:
@p-fernandez

Reply:
It wouldn't hurt but maybe not needed. It would be weird to think a TTL is based in milliseconds, very unuseful cache. I think that later we have some values in milliseconds led me to wonder the unit of this value.